syntax = "proto3";

package content_grpc_package;

service RecoService {
    rpc RecoSys (RecoRequest) returns (RecoResponse) {}
}

message RecoRequest {
    string device_uuid = 1;
    string channel_id = 2;
    int32 request_num = 3;
}

message RecoResponse {
    repeated Content content = 1;
    repeated string content_id_list = 2;
}

message Content {
    string content_id = 1;
    int32 content_type = 2;
    string recall_reason = 3;
}